src/containers/offer-details/offer-carousel/OfferCarousel.tsx

```diff
@@ -41,23 +41,24 @@ const OfferCarousel = ({ offer }: OfferCarouselProps) => {
 
   const slidesToShow = getScreenBasedLimit(breakpoints, itemsLoadLimit)
 
-  const getOffers = useCallback(
-    () =>
-      OfferService.getOffers({
-        authorRole: offer.authorRole,
-        subjectId: offer.subject._id,
-        proficiencyLevel: offer.proficiencyLevel,
-        languages: offer.languages,
-        excludedOfferId: offer._id,
-        status: StatusEnum.Active,
-        limit: 9
-      }),
-    [offer]
-  )
-
-  const { response } = useAxios<GetOffersResponse>({
-    service: getOffers,
-    defaultResponse
+  const getOffers = useCallback(() => {
+    return OfferService.getOffers({
+      authorRole: offer.authorRole,
+      subjectId: offer.subject._id,
+      proficiencyLevel: offer.proficiencyLevel,
+      languages: offer.languages,
+      excludedOfferId: offer._id,
+      status: StatusEnum.Active,
+      limit: 9
+    })
+  }, [offer])
+
+  const { data: response } = useQuery<ItemsWithCount<Offer>>({
```

Comment:
You don't need to type `useQuery`, it's already typed by `getOffers` function
```suggestion
  const { data: response } = useQuery({
```